website/blog/2022-11-30-databend-weekly-70.md

```diff
@@ -0,0 +1,185 @@
+
+**Base**
+
+- remove common macros ([#8936](https://github.com/datafuselabs/databend/pull/8936))
+
+**Format**
+
+- TypeDeserializer get rid of FormatSetting ([#8950](https://github.com/datafuselabs/databend/pull/8950))
+
+**Planner**
+
+- refactor extract or predicate ([#8951](https://github.com/datafuselabs/databend/pull/8951))
+
+**Processors**
+
+- optimize join by merging build data block ([#8961](https://github.com/datafuselabs/databend/pull/8961))
+
+**New Expression**
+
+- allow sparse column id in chunk, redo [#8789](https://github.com/datafuselabs/databend/pull/8789) with a new approach. ([#9008](https://github.com/datafuselabs/databend/pull/9008))
+
+### Documentation :notebook_with_decorative_cover:
+
+- i18n support with crowdin ([#8987](https://github.com/datafuselabs/databend/pull/8987), [#8997](https://github.com/datafuselabs/databend/pull/8997), etc.)
+
+### Bug Fixes :wrench:
+
+**Base**
+
+- try fix lost tracker ([#8932](https://github.com/datafuselabs/databend/pull/8932))
+
+**Meta**
+
+- fix share db bug, create DatabaseIdToName if need ([#9006](https://github.com/datafuselabs/databend/pull/9006))
+
+**Mysql handler**
+
+- fix mysql conns leak ([#8894](https://github.com/datafuselabs/databend/pull/8894))
+
+**Processors**
+
+- try fix update list memory leak ([#9023](https://github.com/datafuselabs/databend/pull/9023))
+
+**Storage**
+
+- read and write block in parallel when compact ([#8921](https://github.com/datafuselabs/databend/pull/8921))
+
+## What's On in Databend
+
+Stay connected with the latest news about Databend.
+
+#### Preview of Infer Schema
+
+You usually need to create a table before loading data from a file stored on a stage or somewhere. Unfortunately, sometimes you might not know the file schema to create the table or are unable to input the schema due to its complexity. 
+
+Introducing the capability to infer schema from an existing file will make the work much easier. You will even be able to query data directly from a stage using a SELECT statement like `select * from @my_stage`.
+
+```sql
+INFER 's3://mybucket/data.csv' FILE_FORMAT = ( TYPE = CSV );
++-------------+---------+----------+
+| COLUMN_NAME | TYPE    | NULLABLE |
+|-------------+---------+----------|
+| CONTINENT   | TEXT    | True     |
+| COUNTRY     | VARIANT | True     |
++-------------+---------+----------+
+```
+
+We've added support for inferring the basic schema from parquet files in [#9043](https://github.com/datafuselabs/databend/pull/9043), and we're now working on [#7211](https://github.com/datafuselabs/databend/issues/7211) to implement `select from @stage`.
+
+**Learn More**
+
+- [PR | add basic schema infer for parquet](https://github.com/datafuselabs/databend/pull/9043)
+- [Issue | query data from S3 location or stage](https://github.com/datafuselabs/databend/pull/7211)
+- [PR | rfc: Infer Schema](https://github.com/datafuselabs/databend/pull/8645)
+
+## Issues
+
+Meet issues you may be interested in and try to solve it.
```

Comment:
We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.